<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 216, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre de cible restante :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nbCible">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInfo">
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonParam">
        <property name="text">
         <string>Paramètre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Louis George Café&quot;;
border-color: rgb(43, 66, 130);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Pour commencer le test, veuillez cliquer que le rond &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#425bff;&quot;&gt;BLEU&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; puis vous devrez cliquez sur le rond &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff6254;&quot;&gt;ROUGE&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; qui apparaitra aléatoirement jusqu'à la fin du test &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuParam_tre">
    <property name="title">
     <string>Paramètre</string>
    </property>
    <addaction name="actionParam_trage"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide ?</string>
    </property>
    <addaction name="actionAide"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_tre"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionParam_trage">
   <property name="text">
    <string>Paramètrages</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
